mathics/builtin/layout.py

```diff
@@ -41,6 +41,78 @@
 SymbolSubscriptBox = Symbol("System`SubscriptBox")
 
 
+def eval_makeboxes_infix(
```

Comment:
Small thing, but instead of `eval_makeboxes_infix` how about more simply `box_infix`.  Right now you can search on `def box_` and you will not find anything.

I think it important that we start being more precise about what we call things, and that I hope will lead to clearer thinking and discussion about these things and lead to less confusion.

Reply:
OK. However, notice the comment that I added in https://github.com/Mathics3/mathics-core/pull/690/commits/11fc371d3d31fb7ae7bfd78883fb4f7bf8fcf976

At some point, this function is going to be removed, in favor of ```mathics.eval.makeboxes.make_boxes_infix```. This function is the one that would be renamed as ```box_infix```.

Reply:
See other comment about the name  `mathics.eval.makeboxes.make_boxes_infix`

Reply:
> See other comment about the name `mathics.eval.makeboxes.make_boxes_infix`

I can't find the other comment right now. Maybe I was interrupted and didn't send. 

Please not `mathics.eval`.  Boxing M-expressions is not the same thing as M-expression evaluation. 

So instead I suggest  `mathics.boxes.box_infix`.

Reply:
Ok, maybe it is going be simpler if you perform the changes over this branch.

Reply:
Ok will do. Thanks.